crates/uv/src/commands/tool/install.rs

```diff
@@ -221,6 +221,10 @@ pub(crate) async fn install(
         }
     };
 
+    if from.name.as_str().eq_ignore_ascii_case("python") {
+        return Err(anyhow::anyhow!("Cannot install Python with `uv tool install`. Did you mean to use `uv python install`?"));
```

Comment:
I would probably make the things within ticks green, for consistency.

Reply:
👍 good call